### PR TITLE
UGENE-6952 GUI tests for "Remote NCBI BLAST..." main menu item

### PR DESCRIPTION
**Pull request by rasputinkirill**

### CODE REVIEW

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp [hunk 1]
```diff
@@ -6445,6 +6445,35 @@ GUI_TEST_CLASS_DEFINITION(test_6941) {
...
 }
 
+GUI_TEST_CLASS_DEFINITION(test_6952) {
+    //1. Click Tools -> BLAST -> Remote NCBI BLAST... Choose _common_data/fasta/human_T1_cutted.fa as input file
+    //2. CLick Next two times, then Run
+    //Expected state: no errors
+    class RemoteBLASTWizardFiller : public CustomScenario {
+    public:
+        void run(HI::GUITestOpStatus &os) override {
+            QWidget *dialog = QApplication::activeModalWidget();
```
Comment (yalgaer) [inline src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp:6455]: nit: Use GTWidget::getActiveModelWidget(os) and you will not need the second line (checking for null)

**Review by yalgaer — COMMENTED** [review]
All is good. Lets wait for the GUI tests to finish.

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp [hunk 2]
```diff
@@ -6445,6 +6445,34 @@ GUI_TEST_CLASS_DEFINITION(test_6941) {
...
+    };
+    GTUtilsWorkflowDesigner::openWorkflowDesigner(os);
+    GTUtilsDialog::waitForDialog(os, new WizardFiller(os, "Remote BLASTing Wizard", new RemoteBLASTWizardFiller()));
+    GTMenu::clickMainMenuItem(os, QStringList() << "Tools"
+                                                << "BLAST"
+                                                << "Remote NCBI BLAST...");
+    
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+    QStringList errors = GTUtilsWorkflowDesigner::getErrors(os);
+    CHECK_SET_ERR(errors.size() == 0, "Unexpected errors");
```
Comment (evelinabis) [inline src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp:6474]: Please check that there are no errors on the dashboard and in the log (if you want to)
```
#include "GTUtilsLog.h"
.
.
.
GUI_TEST_CLASS_DEFINITION(test_6952) {
    const GTLogTracer lt; // defined at the beginning
    .
    .
    .
    CHECK_SET_ERR(!lt.hasErrors(), "Errors in log: " + lt.getJoinedErrorString());
    CHECK_SET_ERR(!GTUtilsDashboard::hasNotifications(os),
                  "Notifications in dashboard: " + GTUtilsDashboard::getJoinedNotificationsString(os));
}
```

**Review by evelinabis — DISMISSED** [review]
Please, run GUI Nightly Ubuntu

**Review by yalgaer — APPROVED** [review]

**Pull request merged**